### DISCUSSION TITLE
Handlebars Templates helper not working in Adaptive cards

### DISCUSSION THREAD

**Discussion by vreezy** [microsoft-search/pnp-modern-search#2977] [Q&A]
**Version used**
4.8

**Describe the bug**
docu references
https://microsoft-search.github.io/pnp-modern-search/extensibility/templating/#handlebars-helpers
https://microsoft-search.github.io/pnp-modern-search/extensibility/templating/#reference-a-slot-in-a-adaptive-card

i want to formate "Created" Date in a AdaptiveCard

**To Reproduce**
Created: 2023-05-11T08:41:42.0000000Z

i think this is the correct way:
```
{
  "type": "TextBlock",
  "text": "${getDate $root.slots['Date'] 'LL'}"
}
```
 result: adaptive card is not rendering

test2:
```
{
  "type": "TextBlock",
  "text": "${getDate ${Created} 'LL'}"
}
```
result: ${getDate 2023-05-11T08:41:42.0000000Z 'LL'}

test3:
```
{
  "type": "TextBlock",
  "text": "<span>{{getDate @root.slots['Date'] 'LL'}}</span>"
}
```
result: {{getDate @root.slots['Date'] 'LL'}}

i testes 100ed other possible ways and nothing works.

**Expected behavior**
i get a date well formated
in handlebars it looks this "May 11, 2023"

** Screenshots
Created is not empty:

![image](https://github.com/microsoft-search/pnp-modern-search/assets/28756163/f80c659f-0f56-45d0-ba0c-5f592baf5f05)

slots.Date is there
![image](https://github.com/microsoft-search/pnp-modern-search/assets/28756163/dacc688a-6ff5-461d-bb0d-26fc74174bef)

**Desktop (please complete the following information):**
 - Browser Chrome

**Comment by vreezy** [comment]
i over see the jPath rule so i tested:
```
{
  "type": "TextBlock",
  "text": "${getDate string(jPath($data, concat('.',$root.slots['Date']))[0]) 'LL'}"
}
```
resut: adaptive card does not renders

**Comment by wobba** [comment]
This is how the default template does it, and should give you a start. Note the `DATE` function which is an adaptive card function.

`"text": "{{DATE(${formatDateTime(string(jPath($data, concat('.',$root.slots['Date']))[0]),'yyyy-MM-ddTHH:mm:ssZ')},LONG)}}",`

**Comment by gretchunkim** [comment]
So close...        Try 

    "value": "${formatDateTime($data.startDate, 'MMM dd yyyy')}"
